### PR TITLE
Refactoring all pages

### PR DESCRIPTION
**Pull request by seuros**
This was in my todo list for long time.  

ping @davydovanton

### CODE REVIEW

**Comment by mperham** [comment]
Whoa, this is awesome!  Thank you, I will test shortly. :heart:

**Comment by davydovanton** [comment]
Nice work! :star2: 
I don't prefer 4 spaces instead 2 but everything else looks fine :+1:

File: js/skq-global.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
     var body = $('html, body');
-    var windowHeight = $(window).height()
-    var scrollPos = document.body.scrollTop
-    var mobileBreak = 992
-
+    var windowHeight = $(window).height();
+    var scrollPos = document.body.scrollTop;
+    var mobileBreak = 992;
+    var logoCarousel = $(".logo-carousel");
+    var navBar = navbar;
```
Comment (mperham) [inline js/skq-global.js]: This breaks for me.

Run `bundle exec jekyll s`, open `http://localhost:4000` and look at the JS console.

**Comment by mperham** [comment]
@seuros Any chance you can look into that broken JS, see comment above?

File: js/skq-global.js (hunk 1 above)
Comment (seuros) [inline js/skq-global.js]: Oh, it should be navBar = $(".navbar"). I will fix it now.

**Comment by mperham** [comment]
Better.  I noticed that the corp logos are cut off.

screen shot 2016-02-06 at 10 55 53

**Comment by mperham** [comment]
The support page is linked incorrectly when on the Pro and Enterprise pages.

**Comment by mperham** [comment]
Images don't work on the footer.  This is pretty buggy still, please do a few minutes of QA and let me know when it's better.

**Comment by mperham** [comment]
You can test the site with `bundle exec jekyll s` and `open http://localhost:4000`

**Comment by seuros** [comment]
Which browser did you use to have the images cut ? It work fine in chrome and firefox.

**Comment by mperham** [comment]
I was using Chrome/OSX.  Really nice work, everything looks good now.

**Pull request merged**